src/frontend/src/components/Header/Header.jsx

```diff
@@ -86,9 +86,19 @@ const Header = () => {
     >
       <List>
         <ListItem className={classes.item}>
-          <Link to="/search" className={classes.links}>
-            Search
-          </Link>
+          <Button color="inherit" size="medium" className={classes.button}>
+            <Link to="/search" className={classes.links}>
```

Comment:
If we have Search in the menu, do we also need it in the header bar for mobile?  I'd suggest it only needs to be in one, not both places.